examples/browser/package.json

```diff
@@ -48,23 +43,19 @@
     "@theia/preferences": "^0.14.0",
     "@theia/preview": "^0.14.0",
     "@theia/process": "^0.14.0",
-    "@theia/python": "^0.14.0",
     "@theia/scm": "^0.14.0",
     "@theia/search-in-workspace": "^0.14.0",
     "@theia/task": "^0.14.0",
     "@theia/terminal": "^0.14.0",
-    "@theia/textmate-grammars": "^0.14.0",
-    "@theia/tslint": "^0.14.0",
     "@theia/typehierarchy": "^0.14.0",
-    "@theia/typescript": "^0.14.0",
     "@theia/userstorage": "^0.14.0",
     "@theia/variable-resolver": "^0.14.0",
     "@theia/workspace": "^0.14.0"
   },
   "scripts": {
     "prepare": "yarn run clean && yarn build",
     "clean": "theia clean && rimraf errorShots",
-    "build": "theiaext compile && theia build --mode development",
+    "build": "theiaext compile && theia build --mode development && yarn postinstall",
```

Comment:
i don't think we should run it on each build, `postinstall` npm hook should be enough